solidity/contracts/isms/ccip-read/TestCccipReadIsm.sol

```diff
@@ -1,14 +1,6 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 pragma solidity >=0.8.0;
```

Comment:
Would be nice if this whole file was moved under `contracts/test`

Reply:
There's a typo in the filename too, there should only be two `C`s.

Reply:
gonna leave this as out of scope